encoding.bs

```diff
@@ -653,33 +654,66 @@ changed, so has the <a>index</a>.
 <var>code point</var> in <var>index</var>, or null if
 <var>code point</var> is not in <var>index</var>.
 
+<div class=note id=visualization>
+<p>There is a non-normative visualization for each index other than index gb18030 ranges.
+index jis0208 also has an alternative Shift_JIS visualization.
+Additionally, there is visualization of the Basic Multilingual Plane coverage of each index other than index gb18030 ranges.
+
+<p>The legend for the visualizations is:
+<ul class=visualizationlegend>
+    <li class=unmapped>Unmapped
+    <li class=mid>Two bytes in UTF-8
+    <li class="mid contiguous">Two bytes in UTF-8, code point follows immediately the code point of previous pointer
+    <li class=upper>Three bytes in UTF-8 (non-PUA)
+    <li class="upper contiguous">Three bytes in UTF-8 (non-PUA), code point follows immediately the code point of previous pointer
+    <li class=pua>Private Use
+    <li class="pua contiguous">Private Use, code point follows immediately the code point of previous pointer
+    <li class=astral>Four bytes in UTF-8
+    <li class="astral contiguous">Four bytes in UTF-8, code point follows immediately the code point of previous pointer
+    <li class=duplicate>Duplicate code point already mapped at an earlier index
+    <li class=compatibility>CJK Compatibility Ideograph
+    <li class=ext>CJK Unified Ideographs Extension A
+</ul>
+</div>
+
 <p>These are the <a lt=index>indexes</a> defined by this
 specification, excluding <a>index single-byte</a>, which have their own table:
 
 <table>
- <tbody><tr><th colspan=2><a>Index</a><th>Notes
+ <tbody><tr><th colspan=4><a>Index</a><th>Notes
  <tr>
   <td><dfn export>index Big5</dfn>
   <td><a href=index-big5.txt>index-big5.txt</a>
+  <td><a href=big5.html>Visualization</a>
```

Comment:
We should add a title here saying "index Big5 visualization". Even better would be to make each link text unique somehow since that affects screen readers.